package.json

```diff
@@ -44,6 +44,7 @@
   "devDependencies": {
     "@angular/compiler-cli": "^0.5.0",
     "@angular/platform-server": "^2.0.0-rc.5",
+    "@types/es6-promise": "0.0.31",
```

Comment:
this is bad, why can't you change the other side to use lib.es2015.promise.d.ts instead?